### PR TITLE
Add emoji and delta for played gamers

### PR DESCRIPTION
**Pull request by DanilinEgor**

### CODE REVIEW

File: Bot.ts [hunk 1]
```diff
@@ -246,15 +245,21 @@ export class Bot {
...
         for (let i = 0; i < gamers.length; ++i) {
           const gamer = gamers[i];
           const line = `${i}. ${gamer.username} - ${gamer.score}`;
-          if (boldUserNames.indexOf(gamer.username) !== -1) {
-            text += '<b>' + line + '</b>';
+          const playedGamer = playedGamers.find(g => g.id == gamer.userId)
+          if (playedGamer !== undefined) {
+            let emoji
+            if (playedGamer.delta > 0) emoji = '🌲'; else emoji = '🔻';
+            text += `<b> ${emoji} ` + line + ` (${playedGamer.delta})</b>`;
```
Comment (vesai) [inline Bot.ts]:
```suggestion
            text += `<b> ${emoji} ${line} (${playedGamer.delta})</b>`;
```

File: Bot.ts [hunk 2]
```diff
@@ -275,3 +280,12 @@ interface UpdateScoresResult {
   winnerScore: number;
   looserScore: number;
 }
+
+export class PlayedGamer {
```
Comment (iKBAHT) [inline Bot.ts]:
```suggestion
interface PlayedGamer {
  id: number;
  delta: number;
}
```

File: Bot.ts [hunk 3]
```diff
@@ -246,15 +245,21 @@ export class Bot {
...
+  ): Promise<string> => {
     return this.db.getGroupGamers(chatId)
       .then(gamers => {
         let text = '';
         for (let i = 0; i < gamers.length; ++i) {
           const gamer = gamers[i];
           const line = `${i}. ${gamer.username} - ${gamer.score}`;
-          if (boldUserNames.indexOf(gamer.username) !== -1) {
-            text += '<b>' + line + '</b>';
+          const playedGamer = playedGamers.find(g => g.id == gamer.userId)
```
Comment (iKBAHT) [inline Bot.ts]: ; in the end

File: Bot.ts [hunk 4]
```diff
@@ -246,15 +245,21 @@ export class Bot {
...
+          if (playedGamer !== undefined) {
+            let emoji
```
Comment (iKBAHT) [inline Bot.ts]:
```suggestion
            const emoji = playedGamer.delta > 0 ? '🌲' ? '🔻';
```

**Review by iKBAHT — CHANGES_REQUESTED** [review]

File: Bot.ts [hunk 5]
```diff
@@ -246,15 +245,21 @@ export class Bot {
...
+            if (playedGamer.delta > 0) emoji = '🌲'; else emoji = '🔻';
```
Comment (vesai) [inline Bot.ts]:
```suggestion
            const emoji = (playedGamer.delta > 0) ? '🌲' : '🔻';
```

**Review by iKBAHT — APPROVED** [review]

**Pull request merged**